apps/site/assets/webpack.config.base.js

```diff
@@ -128,7 +128,10 @@ module.exports = {
   },
 
   plugins: [
-    new CopyWebpackPlugin([{ from: "static/**/*", to: "../../" }], {}),
+    new CopyWebpackPlugin([
+      { from: "static/**/*", to: "../../" },
+      { from: "node_modules/focus-visible/dist/focus-visible.min.js", to: "../js" },
```

Comment:
I see why you want to do this rather than importing it into our existing `app.js`, since we're only planning to load the file if needed but we can't evaluate whether it's needed until we load the page in the browser. Pretty neat approach.